tools/generate-sqlite-files/main.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from collections import defaultdict
+import glob
+import os
+from pathlib import Path
+import pandas as pd
+import re
+import sqlite3
+import tempfile
+import zipfile
+
+# Usage:
+# python main.py --zip all.zip --sqlite file.sqlite3
+parser = argparse.ArgumentParser()
+dtypes = defaultdict(lambda: str)
+
+def main(args):
+    # Name the sqlite the same as the zipfile.
+    sqlitefilename = f'{Path(args.zip).stem}.sqlite3'
+    print(f'Writing to {sqlitefilename}')
+    if os.path.isfile(sqlitefilename):
+        os.remove(sqlitefilename)
+
+    # Do everything in a temp dir.
+    # It will disappear when we hit the end of the with block.
```

Comment:
```suggestion
    # Do everything in a temp dir.
    # It will disappear when we hit the end of the with block.
```
This is a non blocker, but what I would personally like, is to have comments removed from file, and put in the `tools/generate-sqlite-files/readme.md` as code snippets with these comments. 

so in `readme.md`
`with tempfile.TemporaryDirectory('_fac') as tdir:` will create a temp dir for running this.. etc etc.

Reply:
Lets pick this up in an iteration. I have a bunch of questions, actually, as to how we would even work some of this into a GH workflow. That would necessarily drive changes to this. For example, we might want to make it a Django command, so it is part of the FAC app. 

The number of changes coming for this script could be many. Or, we might dump it. So, at that point, we have opportunities to iterate/reshape the whole thing. Which we might have to.